homeassistant/components/device_tracker/icloud.py

```diff
@@ -335,22 +329,46 @@ def determine_interval(self, devicename, latitude, longitude, battery):
                  self._overridestates.get(devicename) == 'away')):
             return
 
+        zones = (self.hass.states.get(entity_id) for entity_id
+                 in sorted(self.hass.states.async_entity_ids('zone')))
+
+        distances = []
+        for zone_state in zones:
+            zone_state_lat = zone_state.attributes['latitude']
+            zone_state_long = zone_state.attributes['longitude']
+            d = distance(
+                latitude, longitude, zone_state_lat, zone_state_long)
+            distances.append(round(d / 1000, 1))
+
+        if len(distances) > 1:
```

Comment:
1. Did you mean `if len(distances) > 0:`?
2. If 1. applies: do `if distances:` instead (fixed pylint issue)

Reply:
Thanks for the catch!